### PR TITLE
fix links in notebooks

### PR DESCRIPTION
**Pull request by hendrik-obj**

### CODE REVIEW

File: notebooks/feature-engineering.ipynb [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "This example shows how Bach can be used for feature engineering. We'll go through describing the data, finding\n",
     "outliers, transforming data and grouping and aggregating data so that a useful feature set is created that\n",
     "can be used for machine learning. We have a separate example available that goes into the details of how a\n",
-    "data set prepared in Bach can be used for machine learning with sklearn [here](https://objectiv.io/modeling/machine_learning/)."
+    "data set prepared in Bach can be used for machine learning with sklearn [here](https://objectiv.io/docs/modeling/machine_learning)."
```
Comment (borft) [inline notebooks/feature-engineering.ipynb:22]: can we add a trailing slash here?

File: notebooks/feature-engineering.ipynb [hunk 2]
```diff
@@ -375,7 +375,7 @@
     "Now that we have our data set, we can use it for machine learning, using for example sklearn. To do so\n",
     "we call `to_pandas()` to get a pandas DataFrame that can be used in sklearn.\n",
     "\n",
-    "Here is our example on how to use Objectiv data and [sklearn](https://objectiv.io/modeling/machine_learning/)."
+    "Here is our example on how to use Objectiv data and [sklearn](https://objectiv.io/docs/modeling/machine_learning)."
```
Comment (borft) [inline notebooks/feature-engineering.ipynb:378]: and here as well

**Review by borft — APPROVED** [review]
lgtm

**Comment by vincenthoogsteder** [comment]
LGTM

**Pull request merged**